### DISCUSSION TITLE
Stream Deck implementation

### DISCUSSION THREAD

**Discussion by rokzin84** [OpenStickCommunity/GP2040-CE#1046] [Ideas]
Hi.
I know keyboard is supported.
But could it be possible to add Stream Deck as a possibility?
Stream Deck meaning ctrl+F button or somehow to make a shortcut for specific functions?

Love the firmware, it has everything I could ever want, besides Stream Deck.
The work you are doing is awesome!!

**Comment by TheTrainGoes** [comment]
Hi @rokzin84 , it is unclear what you are asking for here.  

Are you just looking for a keyboard mode that allows you to do multiple keys assigned to a single GPIO?

Example:

GPIO1 = `CTRL` + `F`

**Comment by rokzin84** [comment]
Exactly what I am looking for.
I have traversed the web interface, but could not find something that did
exactly that, may have been that I have overlooked it :/

tir. 11. jun. 2024, 15:21 skrev TheTrain ***@***.***>:

**Comment by TheTrainGoes** [comment]
@rokzin84 Something we will look into.  We have plans to roll this functionality into the main inputs section and will hopefully be able to bridge that to keyboard as well.
